newm-server/src/main/kotlin/io/newm/server/database/exposed/ExposedArrays.kt

```diff
@@ -0,0 +1,130 @@
+package io.newm.server.database.exposed
```

Comment:
This is interesting. I've always been doing it manually using TEXT fields and then split() and joinToString() to serialize and deserialize to the db. This looks like a nice approach.

Reply:
Yes, the advantage is that we can use queries based on individual value of the array. Like query all songs of a specific genre like "Rock".